common/src/primitives/id.rs

```diff
@@ -29,6 +30,12 @@ impl From<GenericArray<u8, typenum::U32>> for H256 {
     }
 }
 
+impl From<H256> for Uint256 {
```

Comment:
Do we have the conversion to the other way?

Reply:
Uint256 to H256? I didn't need it (yet?), so I didn't do one.
If it's needed, I'll do it

Reply:
In the interest of having full tooling. Please provide both conversions and test them.

Reply:
I'm actually with Carla on this one, even if it's a minor thing. I don't think we should have code that we don't need because it's just dead weight with no concrete benefit.

Reply:
We will need it sooner or later. Even in bitcoin there's both ArithToUint256() and UintToArith256(). Assuming we won't need them is very optimistic. We're now in that mindset, so doing that now will reduce the risk of having to switch context later.

Reply:
I'm with Sam here, having it ready to go on the off chance we need it in the future makes sense.

Reply:
added conversion Uint256to H256